bypros/proxy.go

```diff
@@ -148,6 +148,9 @@ func (s *Service) listenBlocks(conn *websocket.Conn) {
 				// that can happen when we close the connection
 				log.Warnf("failed to read request: %v", err)
 			}
+
+			log.LLvl1("stops listening on blocks")
```

Comment:
```suggestion
			log.Lvl1("stops listening on blocks")
```

Is this really a level 1?

For debugging prints, please use `log.Print`. That is the unofficial `log.Debug`...

Reply:
It's official now:

https://github.com/dedis/cothority/pull/2447/checks?check_run_id=2380240597#step:6:114

;)

Reply:
> Is this really a level 1?

Yes, this log is intended to be there.